agents/sbd/fence_sbd.py

```diff
@@ -469,11 +521,17 @@ def main():
     # we check against the defined timeouts. If the pacemaker timeout is smaller
     # then that defined within sbd we should report this.
     power_timeout = int(options["--power-timeout"])
-    sbd_msg_timeout = get_msg_timeout(options)
+    sbd_msg_timeout, sbd_watchdog_timeout = get_sbd_header_timeout(options)
     if 0 < power_timeout <= sbd_msg_timeout:
         logging.warning("power timeout needs to be \
                 greater then sbd message timeout")
 
+    action = options.get("--action", None)
+    if "--crashdump" in options and action in ("reboot", "off"):
+        crashdump_timeout = get_crashdump_timeout()
+        logging.warning("crashdump option is enabled while doing %s action", action)
+        logging.warning("sbd watchdog timeout: %d, crashdump timeout: %d", sbd_watchdog_timeout, crashdump_timeout)
```

Comment:
maybe 'crashdump timeout (compare to setting on target-node %s): ...' to make it clear that we actually don't know about the relevant setting.

Reply:
Add target node(from reading --plug option) here

Reply:
yeah - target node is interesting in any case although it might be logged in the course of the fencing action already. My suggestion was as well intended to make it obvious which config parameters are synced (watchdog-timeout) or potentially configured individually (crash-timeout, reboot-action, ...) on each node (and thus we advise to compare with settings on the target node).

Reply:
I think the current log already shows enough details.
Recently, I was also working on the interface to set crashdump from crmsh by using such interface `crm sbd configure crashdump-watchdog-timeout=<int>`, then crmsh will make sure all settings are synced between cluster nodes.

Reply:
> maybe 'crashdump timeout (compare to setting on target-node %s): ...' to make it clear that we actually don't know about the relevant setting.

Changed